token/jwk/fetch_keys_blackbox_test.go

```diff
@@ -66,3 +67,15 @@ func TestFetchKeys(t *testing.T) {
 	})
 
 }
+
+func defaultMockTokenManagerConfiguration(t *testing.T) *tokensupport.ManagerConfigurationMock {
```

Comment:
this name looks too verbal `defaultMockTokenManagerConfiguration()` it's feels like java_naming and not very much golang idiomatic.  If not different but can we have `defaultMockTokenMgrConfig()` or something ??

Reply:
@nurali-techie I agree, it's too verbal, but I think it's clear from it's name what it is doing